ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/hub-side-templating/ns.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
No need for this , the NS already defined at https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/ns.yaml#L11